### PR TITLE
Add check on queue submissions and deletion

### PR DESCRIPTION
**Pull request by mcoughlin**
**Does this pull request make any changes to the database?**
No

Code changes that affect the database require special attention for data migration.
If your change affects the database, describe the migration plan here.

### CODE REVIEW

File: growth/too/tasks/scheduler.py [hunk 1]
```diff
@@ -151,13 +151,21 @@ def ping_decam():
...
 def schedule_ztf(json_data):
-    r = requests.put(urllib.parse.urljoin(ZTF_URL, 'queues'),
-                     json={'targets': json_data["targets"],
-                           'queue_name': json_data["queue_name"],
-                           'validity_window_mjd':
-                               json_data["validity_window_mjd"],
-                           'queue_type': 'list'})
-    r.raise_for_status()
+
+    r = requests.get(urllib.parse.urljoin(ZTF_URL, 'queues'), json={})
```
Comment (lpsinger) [inline growth/too/tasks/scheduler.py]: Please add an `r.raise_for_status()` here.

File: growth/too/tasks/scheduler.py [hunk 2]
```diff
@@ -151,13 +151,21 @@ def ping_decam():
...
+    data_all = r.json()
+    queue_names_list = []
+    for data in data_all:
```
Comment (lpsinger) [inline growth/too/tasks/scheduler.py]: `queue_names_list = [d['queue_name'] for d in data_all]`

File: growth/too/views.py [hunk 3]
```diff
@@ -206,6 +206,9 @@ def queue():
             urllib.parse.urljoin(tasks.scheduler.ZTF_URL, 'queues'),
             json={'queue_name': queue_name})
 
+        tasks.email.delete_too(queue_name)
```
Comment (lpsinger) [inline growth/too/views.py]: Please submit these tasks asynchronously using `.delay` or `.apply_async`.

**Comment by mcoughlin** [comment]
@lpsinger ready for more improvements ;)

**Pull request closed without merging**